plugins/vuex-persist.client.js

```diff
@@ -4,6 +4,6 @@ import VuexPersistence from "vuex-persist";
 export default ({ store }) => {
   new VuexPersistence({
     /* your options */
-    modules: ["auth"],
+    modules: ["auth", "setting"],
```

Comment:
U can remove comments
```suggestion
    modules: ["auth", "setting"],
```